cns/restserver/ipam.go

```diff
@@ -76,7 +76,7 @@ func (service *HTTPRestService) releaseIPConfigHandler(w http.ResponseWriter, r
 	}()
 
 	err = service.Listener.Decode(w, r, &req)
-	logger.Request(service.Name, &req, err)
+	logger.Request(service.Name, req, err)
```

Comment:
Can you append operationName to differentiate between requireIpConfig and relaeseIpConfig

Reply:
Sure.